src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultGenRepoInfoFileConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Isn't this calling 'gcloud app gen-repo-info-file`? I don't feel strongly about it staying in this package or not, but just note that stage is also in this package.